lib/matplotlib/backends/backend_pgf.py

```diff
@@ -23,37 +23,52 @@
     _create_pdf_info_dict, _datetime_to_pdf)
 from matplotlib.path import Path
 from matplotlib.figure import Figure
+from matplotlib.font_manager import FontProperties
 from matplotlib._pylab_helpers import Gcf
 
 _log = logging.getLogger(__name__)
 
 
+DOCUMENTCLASS = r"\documentclass{article}"
```

Comment:
Why do you want to pull this out:
- should this really be public?
- do you anticipate to modify this?

Reply:
@timhoffm There may be a good reason for a user to want to modify this: https://github.com/matplotlib/matplotlib/issues/28119

Reply:
@timhoffm Sorry, I missed this comment. The main reason why I pulled it out into a constant is because inconsistent documentclass declarations are precisely what caused the issue that this is fixing. I’m not sure what you mean by “public” – do you want the variable name to start with an underscore?

Reply:
Yes, please use an underscore if it's not meant to be modified by users. We're quite strict on this nowadays because otherwise backward-compatible changes become very hard.

If you want users to give the freedom to change `DOCUMENTCLASS`, then it would need documentation.

Reply:
OK, done!